addons/udes_common/tools/helpers.py

```diff
@@ -0,0 +1,15 @@
+"""Helpers"""
+
+# Fields that will be hidden from the form views if by default will not be needed to be shown.
+# In order to extend should be added the model like a key if doesnt exist yet, adding the fields
+# needed to be hidden and to show a button View More should be added to the form view.
+# Fields that are added in helpers but not in models will not raise any error just skip them, this
+# is done in order not to add dependencies with modules where fields are defined.
+hidden_fields_from_form_view = {
```

Comment:
Could we look into setting the value of the attribute on the relevant model directly?

So by default it could be an empty list.

Then in `udes_stock` we can set this for `stock.picking`.

And for hiding the fields, `self.hidden_fields_from_form_view` should do the trick I think.

Reply:
Liked this suggestion :)

Reply:
Made the suggested changes in both MR-s